### ISSUE TITLE
Eliminating GPS outliers doesn't always work.

### ISSUE THREAD

**Issue by dumbenis** [tumic0/GPXSee#525]
I disabled the "Eliminate GPS outliers" setting in preferences/options to see GPS outliers in GPX files.
In the display of a GPX file, the GPS outliers were not visible. See example: left Avenue, right  GPXSee.
<img width="1268" alt="07174502" src="https://github.com/tumic0/GPXSee/assets/77728140/f7f841f7-0f57-4fd4-867a-fb299d71f4ae">

**Comment by tumic0** [comment]
Please provide the GPS log file, without the data it is not possible to investigate the issue any further.

**Comment by tumic0** [comment]
I have analyzed the file and the main problem is the data is absolute crap. The whole file is full of different GPS positions with the same timestamp which is nonsense by definition. The outlier with the missing first digit (51N -> 1N) is just the icing on the cake.

You do not see the "path to Africa" because the pause detection, not the outlier detection algorithm, eliminates it. All the algorithms in GPXSee are designed to work on real data that may contain "real errors", but systematic errors like "some Chinese crap producer not writing sub-second timestamps when sub-second sampling is used" will break stuff. And that's ok as any attempt to handle even such crap data would almost certainly be at the expense of the "normal usage".

**Comment by dumbenis** [comment]
First of all thank you for your reply. The GPX data is part of a test of the Columbus P-10 Pro Submeter (0.5m) GPS/GNSS Data Logger and USB Receiver with Broadcom BCM47755 GNSS Chip.
I know that with a built-in antenna sub-meter accuracy cannot be achieved.
As an user, I am extensively testing this GPS logger because I experience various problems: the well-known and well-documented problem of outliers, invalid timestamps such as <time> 2064-00-00T10:13:16Z </time> and unable to recover after signal loss. Columbus replaced my unit but I still experience the same problems.
I investigate the test results and report errors to Columbus (Unknown Victory Co., Ltd. with Hotmail e-mail addresses) and their reseller in Germany, because the Columbus P-10 Pro is not a reliable GPS-logger now. 
In my error reports I use the Avenue and GPXSee applications.
I saw the differences in graphic display and submitted a report to you.

You pointed out the problem of systematic errors in the GPX data of "some Chinese crap producer not writing sub-second timestamps when sub-second sampling is used".
In the GPX data, I saw that a GPS coordinate is recorded five times every second. 
The Columbus P-10 Pro tested appears to use the 5Hz log mode, where it records 5 trackpoints per second.
Your conclusion is correct: the Columbus P-10 Pro uses sub-second sampling without sub-second timestamps.
So Columbus implemented sub-second sampling incorrectly.

Columbus highlights the 5Hz log mode in its advertising, but says nothing about the faulty implementation and another drawback: reduced accuracy & increase drift by low velocity scenarios(<20km/h). See gpswebshop.com.
In short: I put the Columbus P-10 Pro in 1Hz log mode and continue testing.

Question: which reliable GPS logger would you recommend?
